x-pack/test_serverless/shared/lib/security/kibana_roles/project_controller_security_roles.yml

```diff
@@ -0,0 +1,670 @@
+# -----
+# Source: https://github.com/elastic/project-controller/blob/main/internal/project/security/config/roles.yml
```

Comment:
do you think these roles are prone to change? do you have an idea how to ensure these definitions are updated when the roles change?

Reply:
They could - for sure - be changed/adjusted. We don't currently have a good way to keep that file in sync with the kibana code, but I have brought up the issue with the Serverless team to determine if we can find a method of doing it automatically.